src/app/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Tabs from './Tabs';
+
+export default {
+  title: 'Component/Tabs',
+  component: Tabs,
+};
+
+export const TabBar = (): JSX.Element => <Tabs />;
```

Comment:
More stories for the different active routes would be great.

```suggestion
export const Home = (): JSX.Element => <MemoryRouter initialEntries={["/"]}><Tabs /></MemoryRouter>;
export const About = (): JSX.Element => <MemoryRouter initialEntries={["/about"]}><Tabs /></MemoryRouter>;
```

With https://reactrouter.com/web/api/MemoryRouter